recipe/meta.yaml

```diff
@@ -31,14 +30,15 @@ requirements:
   host:
     - libcurl  # [not win]
     - bzip2  # [not win]
-    # add back when we re-unvendor zlib
-    # - zlib  # [not win]
+    - zlib  # [not win]
 
 test:
   commands:
     - test -f ${PREFIX}/lib/libcfitsio{{ SHLIB_EXT }}  # [not win]
     - if not exist %LIBRARY_LIB%\cfitsio.lib exit 1  # [win]
     - if not exist %LIBRARY_BIN%\cfitsio.dll exit 1  # [win]
+  downstreams:
+    - gdal
```

Comment:
```suggestion
    - gdal  # [osx]
```

Reply:
We may need to decorate the line above too